src/crypto/bcrypt/password.cr

```diff
@@ -15,6 +15,8 @@ require "../subtle"
 #
 # See `Crypto::Bcrypt` for hints to select the cost when generating hashes.
 class Crypto::Bcrypt::Password
+  SUPPORTED_VERSIONS = {"2", "2a", "2b"}
```

Comment:
Please use an array. Tuple is inappropriate for a potentially expandable list.

Reply:
On a side note, I wrote this based on the following template from existing Crystal source:
```
src/http/common.cr:  SUPPORTED_VERSIONS = {"HTTP/1.0", "HTTP/1.1"}
```

Reply:
That's wrong. Let's not repeat that.

With tuples, the type of the constant would change as soon as a new supported version is added. That's unacceptable.

Reply:
The type is `Enumerable(String)`, and that should be enough. No need for concrete types here as there's no need to extend them.